Codeinwp/neve
# chore: card component header icon when using legacy dashicons [dashboard]

### Summary
Gutenberg plugin seems to have switched the Icon component to the old dashicons approach (using spans with classes) instead of using SVG's. This adapts the Card component from the dashboard for icons that are placed in its' header to look the same as SVG icons.  


### Will affect visual aspect of the product

NO

### Test instructions


- Feedback card icon should be the same as the other card icons with Gutenberg plugin activated
- You can add `return true;` at the top of [this method](https://github.com/codeinwp/neve/blob/993b5ed4a67fbce55b0a7269f730484a90c89e72/dashboard/inc/Main.php#L476) in order to make the card appear.


Ref #2118.
